dbos/_migration.py

```diff
@@ -343,11 +353,34 @@ def get_sqlite_timestamp_expr() -> str:
 ALTER TABLE operation_outputs ADD COLUMN completed_at_epoch_ms BIGINT;
 """
 
+sqlite_migration_six = """
+-- SQLite doesn't support dropping primary keys, so we need to recreate the table
+CREATE TABLE workflow_events_new (
+    workflow_uuid TEXT NOT NULL,
+    key TEXT NOT NULL,
+    value TEXT NOT NULL,
+    function_id INTEGER NOT NULL DEFAULT 0,
+    PRIMARY KEY (workflow_uuid, key, function_id),
+    FOREIGN KEY (workflow_uuid) REFERENCES workflow_status(workflow_uuid)
+        ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+INSERT INTO workflow_events_new (workflow_uuid, key, value, function_id)
+SELECT workflow_uuid, key, value, 0 FROM workflow_events;
+
+DROP TABLE workflow_events;
+
+ALTER TABLE workflow_events_new RENAME TO workflow_events;
+
+ALTER TABLE streams ADD COLUMN function_id INTEGER NOT NULL DEFAULT 0;
+"""
```

Comment:
What about we make this a transaction? sqlite seems to support transactional DML + DLL

Reply:
It already is